### ISSUE TITLE
Deprecation warning when using the plugin

### ISSUE THREAD

**Issue by dwt** [couchtyp/certbot-dns-schlundtech#4]
when running the plugin, I'm getting this warning:

```
Plugin legacy name certbot-dns-schlundtech:auth may be removed in a future version. Please use auth instead.
```

I'm not sure I understand what this is trying to tell me - can you provide me with some pointers as to how to help debug best?

**Comment by couchtyp** [comment]
Thanks for reporting this issue.

It seems certbot changed the way referencing plugins by name is handled.
When specifying a plugin, one should no longer use the fully qualified name (`certbot-dns-schlundtech:auth`) but the plugin name that is specified (see `setup.py` > `certbot.plugins`), which in this case is unfortunately just `auth`.

So, to get rid of the deprecation warning for now, instead of passing `-a certbot-dns-schlundtech:auth` you could use `-a auth`.
`certbot plugins --authenticators` should get you a list of all (authenticator) plugins and their respective names.

I will have a look at the issue in the next few days and hopefully come up with a fix that is backward compatible.

**Comment by dwt** [comment]
No wories, for now it is only a warning (which is why I reported it now so there is time to fix it).

Maybe providing two plugins, i.e. something like:

```python
    entry_points={
        'certbot.plugins': [
            'auth = certbot_dns_schlundtech.auth:Authenticator',
            'certbot_dns_schlundtech:auth = certbot_dns_schlundtech.auth:Authenticator',
        ],
    },
```

is possible? Or something close?

**Comment by couchtyp** [comment]
Turns out it's not that easy...
For example, command line arguments and credential configuration depend on the name of the plugin.

I decided to archive the old version as release v0.03.0 - which is usable, but will in fact fail as soon as the old plugin names are no longer supported - and refactor the project to satisfy the current plugin model.

Good news is that with the approach you suggested the old command line arguments and configuration should still work - for now.
Hopefully providing the plugin under two different names will not cause more issues.
